src/cc_catalog_airflow/dags/provider_api_scripts/phylopic.py

```diff
@@ -0,0 +1,357 @@
+                                     license_url=details[6],
+                                     width=details[4],
+                                     height=details[5],
+                                     creator=details[7],
+                                     title=details[8],
+                                     meta_data=details[9],
+                                     )
+
+        if ctr % LIMIT == 0:
+            # Add 5 images at a time to file
+            image_store.commit()
+            logger.info('Flushed current buffer to file.')
+        ctr += 1
+
+    image_store.commit()
+
+
+def _get_total_images():
+    # Get the total number of PhyloPic images
+    total = 0
+    endpoint = 'http://phylopic.org/api/a/image/count'
+    result = _get_response_json(endpoint=endpoint, retries=2)
+
+    if result and result.get('success') is True:
+        total = result.get('result')
+
+    return total
+
+
+def _get_image_IDs(**args):
+    limit = LIMIT
+    offset = 0
+    endpoint = ''
+
+    if args.get('date'):
+        # Get a list of recently updated/uploaded objects.
+        date = args['date']
+        endpoint = 'http://phylopic.org/api/a/image/list/modified/{}'.format(
+                  date)
+
+    else:
+        # Get all images and limit the results for each request.
+        offset = args['offset']
+        endpoint = 'http://phylopic.org/api/a/image/list/{}/{}'.format(
+                  offset, limit)
+
+    if endpoint == '':
+        logger.warning('Unable to parse endpoint: {}, args: {}'.format(
+                       endpoint, args))
+        return [None]
+
+    result = _get_response_json(endpoint=endpoint, retries=2)
+    image_IDs = []
+
+    if result and result.get('success') is True:
+        data = list(result.get('result'))
+
+        if len(data) > 0:
+            for i in range(len(data)):
+                image_IDs.append(data[i].get('uid'))
+
+    if not image_IDs:
+        logger.warning('No content available!')
+        return [None]
+
+    return image_IDs
+
+
+def _get_meta_data(_uuid):
+    logger.info('Processing UUID: {}'.format(_uuid))
+
+    start_time = time.time()
+    base_URL = 'http://phylopic.org'
+    img_URL = ''
+    thumbnail = ''
+    width = ''
+    height = ''
+    foreign_id = ''
+    foreign_url = ''
+    meta_data = {}
+    creator = ''
+    title = ''
+
+    endpoint = "http://phylopic.org/api/a/image/{}?options=credit+" \
+               "licenseURL+pngFiles+submitted+submitter+taxa+canonicalName" \
+               "+string+firstName+lastName".format(_uuid)
+    request = _get_response_json(endpoint=endpoint, retries=2)
+
+    if request and request.get('success') is True:
+        result = request['result']
+
+    license_url = result.get('licenseURL')
+    img_info = _get_image_info(result)
+
+    if img_info is not None:
+        img_URL = img_info[0]
+        width = img_info[1]
+        height = img_info[2]
+        thumbnail = img_info[3]
```

Comment:
If you refactor by returning a tuple from `_get_image_info` (or even if you don't), this could be:
```python
    img_URL, width, height, thumbnail = _get_image_info(result)
```